### ISSUE TITLE
[E3] Problema con add Reference con el MVC

### ISSUE THREAD

**Issue by agustinbc7** [franciscogazitua/IIC2113-2024#93]
Hola, para implementar mvc, cree un proyecto para el modelo. Ya le hice al controlador el Add reference al modelo, pero el proyecto modelo me sale con error (ningún archivo dentro de este proyecto sale con error, es solo el proyecto). Y al correr los tests, obtengo este error: 0>CSC: Error CS5001 : El programa no contiene ningún método 'Main' estático adecuado para un punto de entrada.
No entiendo bien si me falta hacer algo o porqué debería crear un main o program (y si necesita algo especial en este caso)

**Comment by RodrigoToroIcarte** [comment]
¿Qué tipo de proyecto es tu modelo? ¿Es una librería o proyecto en consola? Me tinca que es proyecto en consola y borraste el Program.cs

**Comment by agustinbc7** [comment]
efectivamente es un proyecto en consola, ¿debo tener un Program.cs aunque no haga nada? porque sí borré ese archivo que venía con un print por defecto

**Comment by agustinbc7** [comment]
Le coloqué un Program.cs que no hace absolutamente nada y se solucionó. ¿No afecta el clean code o sí?

**Comment by RodrigoToroIcarte** [comment]
No. Formalmente lo ideal es crear el proyecto como una librería... porque no queremos que tenga un Program.cs. Pero da un poco lo mismo. No afectará tu nota de clean code.
